kubernetes/windows/main.ps1

```diff
@@ -263,6 +263,27 @@ function Generate-Certificates {
     C:\\opt\\omsagentwindows\\certgenerator\\certificategenerator.exe
 }
 
+function Bootstrap-CACertificates {
+    try {
+        # This is required when the root CA certs are different for some clouds.
+        $caCerts=Invoke-WebRequest 'http://168.63.129.16/machine?comp=acmspackage&type=cacertificates&ext=json' -UseBasicParsing | ConvertFrom-Json
```

Comment:
since this outbound network call, we need to honor the proxy configuration, if its specified similar to other network call we do in this file.

Reply:
@ganga1980 - could you please remove the cloud names from this comment since it is open source?

Reply:
This is the IP address over which the VM talks to the physical server on which the vm is hosted. I dont think this requires proxy settings.